packages/core/src/app/checkout/getCheckoutStepStatuses.ts

```diff
@@ -62,7 +63,7 @@ const getCustomerStepStatus = createSelector(
             return {
                 type: CheckoutStepType.Customer,
                 isActive: false,
-                isComplete: customer?.isStripeLinkAuthenticated !== undefined ?? isComplete,
+                isComplete: paymentProviderCustomer?.authenticationState !== undefined ?? isComplete,
```

Comment:
This operator `??` works only with `undefined` and `null` types. It is not work correctly with any of boolean type. This condition `paymentProviderCustomer?.authenticationState !== undefined ?? isComplete` will always return first value and no matter what it's gonna be: `true` or `false`

Reply:
Thanks @bc-nick I made the change